server/src/main/java/org/elasticsearch/index/IndexModule.java

```diff
@@ -84,8 +84,14 @@
  */
 public final class IndexModule {
 
+    public static final Setting<Boolean> NODE_STORE_ALLOW_MMAPFS =
+            Setting.boolSetting("node.store.allow_mmapfs", true, Property.NodeScope);
+
+    public static final Setting<List<String>> NODE_ALLOWED_INDEX_STORE_TYPES_SETTING =
```

Comment:
I was expecting that this setting would go away?

Reply:
Oh, that is leftover. It is indeed not used anywhere. Sorry for that. I will remove shortly.

Reply:
@jpountz I pushed bb9f1f24fad2a08d2e1bffef81d2f20a7a72704c.